tasks/bloom-etl/src/etl/extract/extractor.ts

```diff
@@ -0,0 +1,72 @@
+import { Axios } from "axios"
+import { Jurisdiction, Listing, Response, UrlInfo } from "../../types"
+import { ExtractorInterface } from "./extractor-interface"
+import { BaseStage } from "../base-stage"
+
+export class Extractor extends BaseStage implements ExtractorInterface {
+  axios: Axios
+  urlInfo: UrlInfo
+  jurisdictions: Array<Jurisdiction>
+
+  constructor(axios: Axios, urlInfo: UrlInfo, jurisdictions: Array<Jurisdiction>) {
+    super()
+    this.axios = axios
+    this.urlInfo = urlInfo
+    this.jurisdictions = jurisdictions
+  }
+
+  private constructEndpoint(id: string) {
+    // Construct endpoint urls that gets base info for all listings for each jurisdiction individually
+    // This enables us to maximize cache hits on the Bloom side
+    const url =
```

Comment:
So just return the string directly rather than putting it in a var?  Yeah, I can do that.  The original implementation concatenated the first part then generated an array of URLs in a loop (hence the bifurcated concat), but that's definitely not needed anymore